R/make_states.R

```diff
@@ -0,0 +1,39 @@
+make_state_map_one <- function(state, geometry = TRUE, epsg = alarm_epsg(state)) {
+
+    nd <- geomander::get_alarm(state = state, geometry = geometry, epsg = epsg)
+
+    nd <- nd %>%
+        summarise(dplyr::across(tidyselect::starts_with("pop"), sum),
+                  dplyr::across(tidyselect::starts_with("vap"), sum),
+                  dplyr::across(tidyselect::matches("_(dem|rep)_"), sum),
+                  dplyr::across(tidyselect::matches("^a[dr]v_"), sum),
+                  geometry = sf::st_union(geometry))
+
+    id_nrv <- stringr::str_detect(colnames(nd), "arv_")
+    id_ndv <- stringr::str_detect(colnames(nd), "adv_")
+
+    nd <- nd %>%
+        mutate(nrv = rowMeans(as.data.frame(nd)[,id_nrv]),
+               ndv = rowMeans(as.data.frame(nd)[,id_ndv]),
+               nrv = round(nrv, 1),
```

Comment:
the other thing is that any references to variables that live inside the data frame `nd` need to be prefixed with `.data`; e.g., `nrv = round(.data$nrv, 1)` . Same on lines 19, 20.